### ISSUE TITLE
SC1087: false positive, array detected but only display of variables

### ISSUE THREAD

**Issue by kyodev** [koalaman/shellcheck#1260]
#### For bugs
- SC1087
- My shellcheck version 0.5.0
- [x ] I tried on shellcheck.net and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions

- [x ] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [x ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash

spc5="value"
script="test"

line="$spc5[$script]"
echo "$line"
```

#### Here's what shellcheck currently says:

```txt
line="$spc5[$script]"
      ^-- SC1087: Use braces when expanding arrays, e.g. ${array[idx]} (or ${var}[.. to quiet).
```

#### Here's what I wanted or expected to see:

```txt
value[test]
```
no error, it is not an array, just a display

**Comment by kyodev** [comment]
sorry, I readed correctly
